### ISSUE TITLE
Latest conda-forge update fails to install latest mamba...

### ISSUE THREAD

**Issue by alphahmed** [mamba-org/mamba#490]
After updating and upgrading conda as usual, trying to install mamba using conda as follows:

`$ conda install mamba -c conda-forge`

Getting:

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /Users/#####/anaconda3

  added / updated specs:
    - mamba


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.8.4                |   py38h32f6830_2         3.0 MB  conda-forge
    libsolv-0.7.14             |       hd174df1_4         403 KB  conda-forge
    mamba-0.1.2                |   py38ha0d09dd_0          94 KB  conda-forge
    python_abi-3.8             |           1_cp38           4 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         3.5 MB

The following NEW packages will be INSTALLED:

  libsolv            conda-forge/osx-64::libsolv-0.7.14-hd174df1_4
  mamba              conda-forge/osx-64::mamba-0.1.2-py38ha0d09dd_0
  python_abi         conda-forge/osx-64::python_abi-3.8-1_cp38

The following packages will be UPDATED:

  conda                       pkgs/main::conda-4.8.4-py38_0 --> conda-forge::conda-4.8.4-py38h32f6830_2


Proceed ([y]/n)? y
```

**Comment by wolfv** [comment]
mamba 0.1.2 is an extremely old version. The latest is 0.5.1
Can you try to explicitly install the latest version with `conda install mamba=0.5.1 -c conda-forge`?
I wonder where the conflict is. 

Also generally it's advised to have a slim base environment (just the most important things in the base env, e.g. conda, mamba, ...) and everything else in new environments.

**Comment by alphahmed** [comment]
After two hours of examining conflicts related to this version specifying command:

`$conda install mamba=0.5.1 -c conda-forge`

The following are the details of the conflicts output:

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
Examining python-language-server: 100%|████████████████████████████ |
Examining conflict for jsonschema entrypoints pydocstyle python-language-server spyder keyring path.py pylint pytest pluggy importlib_metadata importlib-metadata flake8 jupyter_client nbconExamining conflict for jsonschema entrypoints pydocstyle python-language-server spyder keyring path.py pylint pytest pluggy importlib_metadata importlib-metadata flake8 jupyter_client nbconExamining conflict for qt lcms2 pillow libtiff: : 327it [1:11:27,  5.62s/it]                                                                                                                 Examining conflict for navigator-updater pyqt matplotlib nltk patsy anaconda-navigator spyder qtconsole statsmodels scipy scikit-learn pytorch seaborn scikit-image: : 330it [1:11:28,  3.61sExamining conflict for astropy pytest: : 331it [1:11:41,  3.61s/it]                                                                                                                          Examining conflict for mamba gstreamer conda-build dbus pyodbc conda-package-handling python-libarchive-c qt libxslt libxml2 lxml cairo gst-plugins-base libarchive glib pango fontconfig harExamining conflict for mamba gstreamer conda-build dbus pyodbc conda-package-handling python-libarchive-c qt libxslt libxml2 lxml cairo gst-plugins-base libarchive glib pango fontconfig harExamining conflict for conda-build libarchive python-libarchive-c: : 336it [1:11:50,  1.85s/it]                                                                                              failed                                                                                                                                                                           

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - alabaster -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|3.4.*|>=3.7,<3.8.0a0']
  - anaconda-navigator -> python[version='2.7.*|3.5.*|3.6.*|3.4.*']
  - anaconda-project -> python[version='2.7.*|3.5.*|3.6.*|>=3.5,<3.6.0a0|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0']
  - anaconda==2020.07 -> python[version='3.6.10|3.7.7|3.8.3',build='hcff3b4d_2|h7579374_2|hcff3b4d_5']
  - atomicwrites -> python[version='2.7.*|3.5.*|3.6.*|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - attrs -> python[version='2.7.*|3.5.*|3.6.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - autopep8 -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - babel -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|3.4.*|>=3.7,<3.8.0a0']
  - backports -> python[version='2.7.*|3.5.*|3.6.*|>=3.5,<3.6.0a0|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0']
  - backports.functools_lru_cache -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=2.7,<2.8.0a0']
  - bkcharts -> python=3.4
  - bleach -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - brotlipy -> python[version='3.4.*|3.6.9|3.8.*|3.7.*',build='1_73_pypy|2_73_pypy|0_73_pypy']
  - cffi -> python[version='2.7.*|3.5.*|3.6.*|3.6.9|3.6.9|3.6.9|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|3.4.*',build='1_73_pypy|2_73_pypy|0_73_pypy']
  - colorama -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - conda-env -> python[version='2.7.*|3.4.*|3.5.*']
  - contextlib2 -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - cryptography -> python[version='3.4.*|3.6.9|3.8.*|3.7.*|<=3.3',build='1_73_pypy|2_73_pypy|0_73_pypy']
  - decorator -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']
  - defusedxml -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0']
  - diff-match-patch -> python[version='2.7.*|3.4.*|3.5.*|3.6.*']
  - filelock -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - flask -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|3.4.*|>=3.7,<3.8.0a0']
  - gevent -> python[version='3.4.*|3.6.9|3.8.*|3.7.*',build='1_73_pypy|2_73_pypy|0_73_pypy']
  - glob2 -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - html5lib -> python[version='2.7.*|3.5.*|3.6.*|>=3.5,<3.6.0a0|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0']
  - imagesize -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0']
  - intervaltree -> python[version='2.7.*|3.5.*|3.6.*']
  - ipywidgets -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|3.4.*|>=3.7,<3.8.0a0']
  - itsdangerous -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']
  - jdcal -> python[version='2.7.*|3.5.*|3.6.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - jinja2 -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - mkl-service -> python[version='2.7.*|3.5.*|3.6.*|3.4.*']
  - mkl_fft -> python=3.4
  - more-itertools -> python=3.4
  - navigator-updater -> python[version='2.7.*|3.5.*|3.6.*|3.4.*']
  - numpydoc -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - olefile -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0']
  - packaging -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - parso -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - pathtools -> python[version='2.7.*|3.4.*|3.5.*|3.6.*']
  - prometheus_client -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - py -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - pyparsing -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - python-dateutil -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - pytorch -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|3.6.9|3.6.9|3.6.9',build='1_73_pypy|2_73_pypy|0_73_pypy']
  - pytz -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - qtawesome -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - qtconsole -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0|>=3.7,<3.8.0a0|3.4.*']
  - qtpy -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|3.4.*|>=3.7,<3.8.0a0']
  - rope -> python[version='2.7.*|3.5.*|3.6.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - snowballstemmer -> python[version='2.7.*|3.4.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']
  - sortedcollections -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|3.4.*|>=3.7,<3.8.0a0']
  - sphinxcontrib-websupport -> python[version='2.7.*|3.5.*|3.6.*|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0']
  - tblib -> python[version='2.7.*|3.5.*|3.6.*|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.5,<3.6.0a0']
  - testpath -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - toml -> python[version='2.7.*|3.5.*|3.6.*|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - toolz -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0']
  - torchvision -> python[version='2.7.*|3.5.*|3.6.*|3.4.*']
  - werkzeug -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0']
  - xlrd -> python[version='2.7.*|3.5.*|3.6.*|>=3.7,<3.8.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.5,<3.6.0a0']
  - xlsxwriter -> python[version='2.7.*|3.5.*|3.6.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - xmltodict -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0']
  - yapf -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - zict -> python[version='2.7.*|3.5.*|3.6.*|3.4.*|>=3.6,<3.7.0a0|>=2.7,<2.8.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']
  - zope.event -> python[version='2.7.*|3.5.*|3.6.*|>=3.5,<3.6.0a0|3.4.*']

Your python: python=3.8

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

The following specifications were found to be incompatible with a past
explicit spec that is not an explicit spec in this operation (conda):

  - mamba=0.5.1 -> conda[version='>=4.8']

The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions
 
```

Following a long list of details, it seems that there are many conflicts.

**Comment by alphahmed** [comment]
I also have to mention that a fresh install of anaconda on another pc also failed to install the latest version.

**Comment by mariusvniekerk** [comment]
The anaconda metapackage imposes a large number of constraints.  Can you try this in a miniconda based environment?

**Comment by alphahmed** [comment]
Still no luck with miniconda.
It actually gave up after only two trials of flexible solves.
Is there any way of using the source code, or just exporting the latest version to pip so we can install it suing pip?

**Comment by wolfv** [comment]
@alphahmed you should have a clean base environment, only containing conda & mamba, and install other packages (including the large anaconda distribution) in environments.

**Comment by wolfv** [comment]
@kelvinlee200113 how many packages are in your base enviornment?

**Comment by stephenshank** [comment]
@kelvinlee200113 I experienced the same issue today with a clean Anaconda install. A clean Miniconda install (which I now see is recommended in the README) allowed me to install mamba.

**Comment by mountaindust** [comment]
I'd like to ping this again: I just installed Anaconda Python 3.9. With that clean install, the first thing I did was install mamba from conda-forge. The version I got was 0.7.6. Attempting mamba install mamba==0.22.1 gets me the message "Encountered problems while solving. Problem: nothing provides requested mamba 0.22.1". That's odd, because that's the exact version listed at https://anaconda.org/conda-forge/mamba.
